src/Tribe/RSVP.php

```diff
@@ -918,8 +918,10 @@ public function save_ticket( $post_id, $ticket, $raw_data = array() ) {
 			return false;
 		}
 
-		// Updates if we should show Description
+		// Updates if we should show Type and Description
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: ddbef4e0c617c6f99c168f9d4d636b96</sub>

Reply:
Let's address this.

Reply:
```suggestion
		// Updates if we should show Type and Description.
```